### PR TITLE
Update eslint dependencies to match zeal config

### PR DESCRIPTION
**Pull request by frank-west-iii**
We need to pin the versions to match eslint-config-zeal otherwise the
new rules implemented in our config are not recognized by the eslint
version used by react-scripts and throws an error.

### CODE REVIEW

File: packages/react-scripts/package.json [hunk 1]
```diff
@@ -34,17 +34,17 @@
     "css-loader": "0.28.7",
     "dotenv": "4.0.0",
     "dotenv-expand": "4.2.0",
-    "eslint": "4.10.0",
-    "eslint-config-prettier": "^2.3.0",
+    "eslint": "4.19.0",
+    "eslint-config-prettier": "^2.9.0",
```
Comment (randycoulman) [inline packages/react-scripts/package.json:38]: Should we take the `^` off of this one too?

File: packages/react-scripts/package.json [hunk 2]
```diff
@@ -34,17 +34,17 @@
...
     "eslint-config-react-app": "^2.1.0",
-    "eslint-config-zeal": "^1.5.0",
+    "eslint-config-zeal": "1.5.0",
     "eslint-import-resolver-webpack": "0.8.1",
     "eslint-loader": "1.9.0",
     "eslint-plugin-flowtype": "2.39.1",
-    "eslint-plugin-import": "2.8.0",
-    "eslint-plugin-jest": "20.0.3",
+    "eslint-plugin-import": "2.9.0",
+    "eslint-plugin-jest": "21.15.0",
```
Comment (randycoulman) [inline packages/react-scripts/package.json:45]: Do we need to upgrade `jest*` as well, or does this work without doing that?

**Review by randycoulman — APPROVED** [review]
A couple of questions, but LGTM!

File: packages/react-scripts/package.json (hunk 2 above)
Comment (frank-west-iii) [reply packages/react-scripts/package.json:45]: It works, but I'll update it for good measure. No harm there.

File: packages/react-scripts/package.json (hunk 1 above)
Comment (frank-west-iii) [reply packages/react-scripts/package.json:38]: I left these matching what we were doing in our config.

File: packages/react-scripts/package.json (hunk 2 above)
Comment (frank-west-iii) [reply packages/react-scripts/package.json:45]: On second thought I think I am going to break that out into another chore. There are probably more dependencies we might want to update.

**Pull request merged**